ee/tabby-webserver/src/cron/mod.rs

```diff
@@ -22,34 +50,41 @@ pub async fn run_cron(
     job: Arc<dyn JobService>,
     worker: Arc<dyn WorkerService>,
     local_port: u16,
-) {
+) -> Option<CronEvents> {
```

Comment:
```suggestion
) -> CronEvents {
```

Reply:
This function can fail in various ways and return early

Reply:
Then let's use Result<CronEvents>